packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx

```diff
@@ -31,16 +31,17 @@ export default function MessageInput({
       <div className="dt-flex dt-flex-col dt-pb-2 dt-mb-2">
         <form onSubmit={onSubmit}>
           <div className="dt-relative">
-            <div className="dt-text-sm dt-break-words dt-py-1 dt-pl-2 dt-pr-11">
+            <div style={{ width: '24rem'}} className="dt-text-sm dt-break-words dt-py-1 dt-pl-2 dt-pr-11">
               {text || 'h'}
             </div>
             <div className="dt-absolute dt-top-0 dt-w-full dt-h-full dt-flex dt-flex-grow dt-items-center">
               <Textarea
+                style={{ width: '29rem'}}
```

Comment:
This is the default chat width of the parent container but for some reason it got overwritten here.